vllm/core/block/prefix_caching_block.py

```diff
@@ -157,6 +158,8 @@ def allocate_mutable(self, prev_block: Block) -> Block:
                 block_id=block_id,
             )
             assert block.content_hash is None
+            block.computed = False
```

Comment:
is this necessary? the prefix caching block constructor will set it to False

Reply:
I see, would remove it